deps/v8/include/v8.h

```diff
@@ -3177,12 +3177,13 @@ class FunctionCallbackInfo {
                           Local<Function> Callee() const);
   V8_INLINE Local<Object> This() const;
   V8_INLINE Local<Object> Holder() const;
+  V8_INLINE Local<Value> NewTarget() const;
```

Comment:
nit: It might be a good idea to add the a DCHECK in `IsConstructCall` to make sure that the equivalence relationship between `is_construct_call_ & 1` and `!NewTarget->IsUndefined` holds.